packages/@aws-cdk/aws-glue-alpha/lib/table.ts

```diff
@@ -73,6 +74,18 @@ export enum TableEncryption {
   CLIENT_SIDE_KMS = 'CSE-KMS'
 }
 
+export interface StorageParameter {
```

Comment:
Move to `storage-parameter.ts`

Reply:
This has been removed, and now is stored in `storage-parameter.ts`